docs/08-bootstrapping-kubernetes-controllers.md

```diff
@@ -82,14 +82,13 @@ ExecStart=/usr/local/bin/kube-apiserver \\
   --authorization-mode=Node,RBAC \\
   --bind-address=0.0.0.0 \\
   --client-ca-file=/var/lib/kubernetes/ca.pem \\
-  --enable-admission-plugins=Initializers,NamespaceLifecycle,NodeRestriction,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota \\
-  --enable-swagger-ui=true \\
+  --enable-admission-plugins=NamespaceLifecycle,NodeRestriction,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota \\
```

Comment:
In terms of Kubernetes beyond the original guide's use of 1.12: 

'Initializers' was deprecated here: https://github.com/kubernetes/kubernetes/pull/72972